### PR TITLE
httphandler: tss api endpoints

### PR DESCRIPTION
**Pull request by gouthamp-stellar**
### What

TSS API endpoints to

1. submit transactions
2. get transaction 

### Why

For clients of TSS to be able to submit transactions and get a transaction that they submitted

### Known limitations

N/A

### Issue that this PR addresses

https://app.asana.com/0/1207947010297920/1208138733594341

#### PR Structure

- [x] It is not possible to break this PR down into smaller PRs.
- [x] This PR does not mix refactoring changes with feature changes.
- [x] This PR's title starts with name of package that is most changed in the PR, or `all` if the changes are broad or impact many packages.

#### Thoroughness

- [x] This PR adds tests for the new functionality or fixes.
- [x] All updated queries have been tested (refer to [this](https://stackoverflow.com/a/29163465) check if the data set returned by the updated query is expected to be same as the original one).

#### Release

- [x] This is not a breaking change.
- [x] This is ready to be tested in development.
- [x] The new functionality is gated with a feature flag if this is not ready for production.

### CODE REVIEW

File: internal/serve/httphandler/tss_handler.go [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+type TSSHandler struct {
+	Router            router.Router
+	Store             store.Store
+	AppTracker        apptracker.AppTracker
+	NetworkPassphrase string
+}
+
+type Transaction struct {
+	Operations []string `json:"operations"`
```
Comment (CaioTeixeira95) [inline internal/serve/httphandler/tss_handler.go:24]: I'd recommend we add `validate` flag here too.

File: internal/serve/httphandler/tss_handler_test.go [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+		Router:            &mockRouter,
+		Store:             store,
+		AppTracker:        &mockAppTracker,
+		NetworkPassphrase: "testnet passphrase",
+	}
+
+	endpoint := "/tss/transactions"
+
+	r := chi.NewRouter()
+	//r.Get("/tss/{transactionhash}", handler.GetTransaction)
```
Comment (CaioTeixeira95) [inline internal/serve/httphandler/tss_handler_test.go:158]: Should we remove this line?

File: internal/serve/httphandler/tss_handler_test.go [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+		resp := rw.Result()
+		respBody, err := io.ReadAll(resp.Body)
+		require.NoError(t, err)
+		var getTxResp GetTransactionResponse
+		_ = json.Unmarshal(respBody, &getTxResp)
+
+		assert.Equal(t, http.StatusOK, resp.StatusCode)
+		assert.Empty(t, getTxResp.Hash)
+		assert.Empty(t, getTxResp.XDR)
+		assert.Empty(t, getTxResp.Status)
```
Comment (CaioTeixeira95) [inline internal/serve/httphandler/tss_handler_test.go:209]: I think we could return 404 instead of 200 when the transaction hash doesn't exist.

**Review by CaioTeixeira95 — APPROVED** [review]
Overall, LGTM.

File: internal/serve/httphandler/tss_handler.go [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+	httpErr := DecodePathAndValidate(ctx, r, &reqParams, t.AppTracker)
+	if httpErr != nil {
+		httpErr.Render(w)
+		return
+	}
+	tx, err := t.Store.GetTransaction(ctx, reqParams.TransactionHash)
+	if err != nil {
+		httperror.InternalServerError(ctx, "unable to get transaction", err, nil, t.AppTracker).Render(w)
+		return
+	}
```
Comment (gsoarescosta) [inline internal/serve/httphandler/tss_handler.go:111]: Just wondering, should we log the transaction hash explicitly here?

File: internal/serve/httphandler/tss_handler.go (hunk 1 above)
Comment (gouthamp-stellar) [reply internal/serve/httphandler/tss_handler.go]: Done

File: internal/serve/httphandler/tss_handler_test.go (hunk 2 above)
Comment (gouthamp-stellar) [reply internal/serve/httphandler/tss_handler_test.go]: Done

File: internal/serve/httphandler/tss_handler_test.go (hunk 3 above)
Comment (gouthamp-stellar) [reply internal/serve/httphandler/tss_handler_test.go]: Done

File: internal/serve/httphandler/tss_handler.go (hunk 4 above)
Comment (gouthamp-stellar) [reply internal/serve/httphandler/tss_handler.go]: Done

**Pull request merged**